odoo/addons/base/tests/test_mail.py

```diff
@@ -34,24 +34,6 @@ def test_basic_sanitizer(self):
             html = html_sanitize(content)
             self.assertEqual(html, expected, 'html_sanitize is broken')
 
-    def test_mako(self):
```

Comment:
Would check that sanitize works on inline_template. Update tests instead of removing them.

Reply:
Yeah, we have enough tests for template themselves I suppose. Ok.